bson/doc.go

```diff
@@ -19,24 +20,96 @@
 // 		i32, ok := val.Int32OK()
 // 		// do something with i32...
 //
-// The D family of types is used to build concise representations of BSON using native Go types.
-// These types do not support automatic lookup.
+// Native Go Types
+//
+// This D and M types defined in this package can be used to build concise representations of BSON using native Go
+// types.
 //
 // Example:
```

Comment:
Since this section is for D and M, add an example of M as well.

Reply:
done.